less/common/Checkbox.less

```diff
@@ -89,34 +89,34 @@
   }
   &:before {
     content: " ";
-    background: @body-bg;
+    background: var(--body-bg);
     border-radius: 11px;
-    box-shadow: 0 2px 4px @shadow-color;
+    box-shadow: 0 2px 4px var(--shadow-color);
   }
 }
 .Checkbox-display {
   font-size: 14px;
 
   .on & {
-    color: #58a400;
+    color: var(--switch-on-color);
   }
   .off & {
-    color: #d0021b;
+    color:var(--switch-off-color);
   }
   .disabled & {
-    color: @muted-more-color !important;
+    color: var(--muted-more-color) !important;
   }
 }
 .Checkbox--switch.loading {
-  color: @muted-more-color !important;
+  color: var(--muted-more-color) !important;
 
   &.on .Checkbox-display {
-    background: fade(#58a400, 0.5);
+    background: transparent;
   }
   &.off .Checkbox-display {
-    background: fade(@control-bg, 0.5);
+    background: transparent;
```

Comment:
Are these changes negligible?

Reply:
yea, I've come across 3 similar instances at least, where the alpha channel is nearly 0, which doesn't really make sense, because that's not something the eye can really see, so it might as well be set to transparent.